prometheus/0.7.2/artifacthub-pkg.yml

```
@@ -17,42 +17,26 @@ links:
   url: https://github.com/keptn-contrib/prometheus-service
 install: '## Execute the following steps to install prometheus-service:
 
-  ### Download the Keptn''s Prometheus service manifest
-
-  ```bash
-
-  wget https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.6.1/deploy/service.yaml
-
-  ```
-
-  ### Replace the environment variable value according to the use case and apply the
-  manifest
-
   ```bash
-
-  kubectl apply -f service.yaml
-
+  helm install -n keptn prometheus-service https://github.com/keptn-contrib/prometheus-service/releases/download/<VERSION>/prometheus-service-<VERSION>.tgz
   ```
 
-  ### Install Role and Rolebinding to permit Keptn''s prometheus-service for performing
-  operations in the Prometheus installed namespace.
+  ### Install Role and RoleBinding to permit prometheus-service for performing operations in the Prometheus installed namespace:
 
   ```bash
-
-  kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.6.1/deploy/role.yaml
-  -n <PROMETHEUS_NS>
-
+  kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-<VERSION>/deploy/role.yaml -n <PROMETHEUS_NS>
```

Encountered an issue with this earlier today (didn't have time to file an issue yet). The role,yaml is not available at that URL. What we probably want is 
```suggestion
  kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/<VERSION>/deploy/role.yaml -n <PROMETHEUS_NS>
```

---

Thanks, that also needs to be udpated on prometheus-service itself. will do that.